### PR TITLE
Implements questions.yaml

### PR DESCRIPTION
**Pull request by Sean-McQ**
requires additional changes to namespace selector filtering - intermediate changes still included in this PR as it fixes some issues but not all

### CODE REVIEW

File: edit/helm.cattle.io.projecthelmchart.vue
Comment (codyrancher) [inline edit/helm.cattle.io.projecthelmchart.vue:33]: Is there a way to pull this information dynamically? I imagine changes will have to be made when repos and version change.

**Review by codyrancher — CHANGES_REQUESTED** [review]
I'm getting a `Repo not found` error when attempting to test this one.

image

I'll take a look again once you fix it.

File: edit/helm.cattle.io.projecthelmchart.vue
Comment (Sean-McQ) [reply edit/helm.cattle.io.projecthelmchart.vue:33]: You're right on both fronts in that it'll need to be updated when prometheus-federator hits the rancher chart repo and that there should be some way for versionName to be dynamic... still not sure why it's not loading at all for you though. I'll look into all three of these.

Comment (Sean-McQ) [reply edit/helm.cattle.io.projecthelmchart.vue:33]: oh! I'm a dummy, 'prometheus-federator' is the arbitrary name I gave the repo when I created it which you wouldn't have done. That won't be a problem once Arvind merges his charts into the rancher chart repo. Probably best to hold off on this one until he makes that change and I can adjust my code.

**Review by codyrancher — APPROVED** [review]

**Pull request merged**
